### ISSUE TITLE
Masking?

### ISSUE THREAD

**Issue by allelopath** [jliljebl/flowblade-forum#74]
Is it possible to mask a video. I have a video that is 1920x1080 but I want to use only 1440x810 (measured from the lower right hand corner).

**Comment by jliljebl** [comment]
Yes, below is how you mask video. (if you want to use part of  video and zoom it larger use *Transform ->Position Scale* filter)

![mask](https://user-images.githubusercontent.com/1823444/86399546-1ae98900-bcb0-11ea-8e03-1b2f4ca443c6.png)

1. Put clip on Track V1
2. Do *Project -> Create Color Clip...* and create black color clip.
3. Color clip on Track V2
5. Add (any) Compositor on color clip.
4. Add filter *Alpha Shape* on color clip, set *Operator: Subtract* use sliders to control mask.

**Comment by allelopath** [comment]
Thanks for your reply.
For step 5, I see the Operator:Subtract on your screenshot, but it is not there on my screen.
Yes I've scrolled down.

Also, the black rectangle I have has fuzzy edges.

Flowblade 1.12.0
Ubuntu 18.0.4

**Comment by jliljebl** [comment]
That was added for 2.2, Flowblade 1.12 is ancient by now.

For Ubuntu 18.04 you can maybe install Flatpak version 2.4, https://github.com/jliljebl/flowblade/blob/master/flowblade-trunk/docs/INSTALLING.md#installing-flatpak-from-flathub

Problem is Ubuntu is not very good at supporting Flatpak installs.

**Comment by allelopath** [comment]
Done!

I installed Flatpak and a bunch of stuff it needed. Then I installed Flowblade 2.4.0.
Then followed your original instructions and I now have the masked video.
Thanks!
